package.json

```diff
@@ -20,6 +20,9 @@
   "dependencies": {
     "body-parser": "^1.18.1",
     "express": "^4.15.4",
-    "nodemon": "^1.12.1"
+    "express-request-proxy": "^2.0.0",
+    "fs": "0.0.1-security",
+    "nodemon": "^1.12.1",
```

Comment:
You do not need `nodemon` as a dependency. It should be installed globally on your computer.